plugins/woocommerce-admin/client/customer-effort-score-tracks/product-mvp-ces-footer.tsx

```diff
@@ -0,0 +1,167 @@
+			select( OPTIONS_STORE_NAME );
+
+		const action = getOption(
+			PRODUCT_MVP_CES_ACTION_OPTION_NAME
+		) as string;
+
+		const shownForActions =
+			( getOption( SHOWN_FOR_ACTIONS_OPTION_NAME ) as string[] ) || [];
+
+		const adminInstallTimestamp =
+			( getOption( ADMIN_INSTALL_TIMESTAMP_OPTION_NAME ) as number ) || 0;
+
+		const allowTrackingOption =
+			getOption( ALLOW_TRACKING_OPTION_NAME ) || 'no';
+
+		const resolving =
+			! hasFinishedResolution( 'getOption', [
+				SHOWN_FOR_ACTIONS_OPTION_NAME,
+			] ) ||
+			! hasFinishedResolution( 'getOption', [
+				PRODUCT_MVP_CES_ACTION_OPTION_NAME,
+			] ) ||
+			adminInstallTimestamp === null ||
+			! hasFinishedResolution( 'getOption', [
+				ADMIN_INSTALL_TIMESTAMP_OPTION_NAME,
+			] ) ||
+			! hasFinishedResolution( 'getOption', [
+				ALLOW_TRACKING_OPTION_NAME,
+			] );
+
+		return {
+			cesShownForActions: shownForActions,
+			allowTracking: allowTrackingOption === 'yes',
+			storeAgeInWeeks: getStoreAgeInWeeks( adminInstallTimestamp ),
+			cesAction: action,
+			resolving,
+		};
+	} );
+
+	const shareFeedback = () => {
+		setShowFeedbackModal( true );
+		updateOptions( {
+			[ SHOWN_FOR_ACTIONS_OPTION_NAME ]: [
+				cesAction,
+				...cesShownForActions,
+			],
+		} );
+	};
+
+	const onDisablingCES = () => {
+		updateOptions( {
+			[ PRODUCT_MVP_CES_ACTION_OPTION_NAME ]: 'hide',
+		} );
+	};
+
+	const recordScore = ( score: number, comments: string ) => {
+		recordEvent( 'ces_feedback', {
+			action: cesAction,
+			score,
+			comments: comments || '',
+			store_age: storeAgeInWeeks,
+		} );
+		createSuccessNotice(
+			__(
+				"Thanks for the feedback. We'll put it to good use!",
+				'woocommerce'
+			),
+			{
+				type: 'snackbar',
+				icon: <span>🌟</span>,
+			}
+		);
+		setShowFeedbackModal( false );
+	};
+
+	const showCESFooter =
+		! isLoading && allowTracking && cesAction && cesAction !== 'hide';
+
+	return (
+		<>
+			{ showCESFooter && (
+				<WooFooterItem>
+					<div className="woocommerce-product-mvp-ces-footer">
+						<div className="woocommerce-product-mvp-ces-footer__container">
+							<Pill>{ __( 'BETA', 'woocommerce' ) }</Pill>
+							{ __(
+								"You're using the new product editor (currently in development). How is your experience so far?",
+								'woocommerce'
+							) }
+							<Button
+								variant="secondary"
+								onClick={ shareFeedback }
+							>
+								{ __( 'Share feedback', 'woocommerce' ) }
+							</Button>
+							<Button
+								onClick={ onDisablingCES }
+								variant="tertiary"
+							>
+								{ __( 'Turn it off', 'woocommerce' ) }
+							</Button>
+						</div>
+						<Button
+							className="woocommerce-product-mvp-ces-footer__close-button"
+							icon={ closeSmall }
+							label={ __(
+								'Remove share feedback',
+								'woocommerce'
+							) }
+							onClick={ onDisablingCES }
+						></Button>
+					</div>
+				</WooFooterItem>
+			) }
+			{ showFeedbackModal && (
+				<CustomerFeedbackModal
```

Comment:
Since I just merged the [CES: Add additional question](https://github.com/woocommerce/woocommerce/pull/35680) PR, we'll want to add the additional props here and double-check that it's working properly after a rebase on trunk. [Here's an example.](https://github.com/woocommerce/woocommerce/blob/28d7ca306bc761bb3b8a2db57f2d59451f977d80/packages/js/customer-effort-score/src/customer-effort-score.tsx#L96)

Reply:
Addressed here: https://github.com/woocommerce/woocommerce/pull/35652/commits/66ad4d08988dbf1e63d8acfd5812dbaf32bf51e0